internal/bitwarden/webapi/client.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	defaultRequestTimeout = 10 * time.Second
+	defaultRequestTimeout = 120 * time.Second
```

Comment:
The timeout includes all retries that might happen because of rate limiting, in which case a request is likely to take more than 60 seconds.